### PR TITLE
Link Component

### PR DESCRIPTION
**Pull request by serling**

### CODE REVIEW

File: components/link/link.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  children,
+  theme,
+  isEmail,
+  className,
+  shouldOpenInNewTab
+}) => (
+  <a
+    className={cn("link", `link--theme-${theme}`, className)}
+    href={isEmail ? `mailto:${href}` : href}
+    target={shouldOpenInNewTab ? "_blank" : undefined}
```
Comment (asbjornh) [inline components/link/link.jsx]: When opening in new tab always add `rel="noopener noreferrer"`. Read about it [here](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)

File: components/link/link.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  href,
+  children,
+  theme,
+  isEmail,
+  className,
+  shouldOpenInNewTab
+}) => (
+  <a
+    className={cn("link", `link--theme-${theme}`, className)}
+    href={isEmail ? `mailto:${href}` : href}
```
Comment (asbjornh) [inline components/link/link.jsx]: I'm not really sure about this, but this seems a little bit specific and like something that could be handled by a backend. What do you think?

File: components/link/link.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.link {
+  text-decoration: none;
+
+  &--theme-underlined {
+    text-decoration: underline;
+  }
+
+  &--theme-default {
+    &:hover {
+      color: red;
```
Comment (asbjornh) [inline components/link/link.scss]: This is unnecessary I think

File: components/link/link.jsx (hunk 1 above)
Comment (serling) [reply components/link/link.jsx]: Ouch. Good call, fixing!

File: components/link/link.jsx (hunk 2 above)
Comment (serling) [reply components/link/link.jsx]: Totally agree. Removing.

File: components/link/link.scss (hunk 3 above)
Comment (serling) [reply components/link/link.scss]: removed!

**Pull request merged**